CoreObjectModel/MutableMetadataModel/Copier.cs

```diff
@@ -3306,9 +3306,9 @@ private void SubstituteElements(Dictionary<ITypeReference, IEnumerable<ICustomAt
       Contract.Requires(!(assembly is Dummy));
       Contract.Ensures(Contract.Result<Assembly>() != null);
 
-      this.TraverseAndPopulateDefinitionCacheWithCopies.Traverse(assembly); //Does not traverse the types
-      foreach (var type in assembly.GetAllTypes())
-        this.TraverseAndPopulateDefinitionCacheWithCopies.Traverse(type);
+      this.TraverseAndPopulateDefinitionCacheWithCopies.Traverse(assembly); // Traverses the types, except for the <Module> type
+      var moduleType = new List<INamedTypeDefinition>(assembly.GetAllTypes())[0];
```

Comment:
Actually it might be easiest to keep the old code and just put a break in the foreach loop body :-).